aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/SamplingRuleApplier.java

```diff
@@ -175,7 +175,7 @@ boolean matches(String name, Attributes attributes, Resource resource) {
     }
 
     return urlPathMatcher.matches(httpTarget)
```

Comment:
What about updating this matcher to HTTP URL instead of target? Separate PR?

Reply:
Yeah separate PR